flume-ng-core/src/test/java/org/apache/flume/source/TestSyslogUdpSource.java

```diff
@@ -301,8 +301,10 @@ public void testClientHeaders() throws IOException {
 
     Map<String, String> headers = e.getHeaders();
 
-    checkHeader(headers, testClientIPHeader, InetAddress.getLoopbackAddress().getHostAddress());
-    checkHeader(headers, testClientHostnameHeader, InetAddress.getLoopbackAddress().getHostName());
+    InetAddress loopbackAddress = InetAddress.getLoopbackAddress();
```

Comment:
At TravisCI `InetAddress.getLoopbackAddress().getHostName()` returns `127.0.0.1`. 
Inet**Socket**Address.getHostName() returns `localhost`.
I.e. the test uses `SyslogUtils.getHostname(SocketAddress)` as SyslogUdpSource does.